frontend/src/components/Credentials/GSecretDialog.vue

```diff
@@ -298,6 +308,9 @@ export default {
     relatedShootCount () {
       return calculateRelatedShootCount(this.shootList, this.binding)
     },
+    orphaned () {
+      return !isSharedCredential(this.binding) && this.binding._secret === undefined && this.binding._workloadIdentity === undefined
```

Comment:
implementation should be aligned with isOrphaned from utils (`const isOrphaned = !_isSharedCredential && !hasOwnSecret && !hasOwnWorkloadIdentity`)